.circleci/config.yml

```diff
@@ -59,36 +61,42 @@ x-integration-steps: &integration-steps
     - store_artifacts:
         path: /tmp/screenshots
 
+x-simple-tests: &simple-tests
```

Comment:
We could move this back to the job now since the thing we tried with this didn't work.

Reply:
I actually think it reads nicer as an inclusion like this. Makes everything at the bottom neat and easy to read. What do you think?

Reply:
Sure - sounds good